src/Commands/make/CmdCommand.php

```diff
@@ -3,6 +3,7 @@
 namespace Blacksmith\Commands\Make;
 
 use Blacksmith\Command;
+use Exception;
```

Comment:
This is temporary. We will need to start creating custom Exceptions. I wanted to know where you wanted to organize custom exceptions first.